eventuate-common-micronaut-data-jdbc/src/main/java/io/eventuate/common/micronaut/data/jdbc/EventuateMicronautDataJdbcStatementExecutor.java

```diff
@@ -5,11 +5,13 @@
 import io.eventuate.common.jdbc.EventuateRowMapper;
 import io.eventuate.common.jdbc.EventuateSqlException;
 import io.micronaut.data.jdbc.runtime.JdbcOperations;
+import org.apache.commons.lang.NotImplementedException;
```

Comment:
Not used?

Reply:
No, at first I did not implement the micronaut version. I will remove.